packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -69,7 +69,7 @@ export const getLogRowStyles = stylesFactory((theme: GrafanaTheme, logLevel?: Lo
       vertical-align: top;
 
       &:hover {
-        .${context} {
+        .log-row-context {
```

Comment:
You can move the variable context to line 64 then, can't you?

Reply:
Yes, good point! On it.

Reply:
You meant it like this, right? https://github.com/grafana/grafana/pull/31026/commits/487b04e09ed602cf923ea13165e9a4f5f1757b38

Reply:
yes